benchmarks/GraphBLAS-sharp.Benchmarks/Algorithms/PageRank.fs

```diff
@@ -0,0 +1,132 @@
+﻿namespace GraphBLAS.FSharp.Benchmarks.Algorithms.PageRank
+
+open System.IO
+open BenchmarkDotNet.Attributes
+open GraphBLAS.FSharp
+open GraphBLAS.FSharp.IO
+open Brahma.FSharp
+open Microsoft.FSharp.Core
+open GraphBLAS.FSharp.Objects.ArraysExtensions
+open GraphBLAS.FSharp.Benchmarks
+open GraphBLAS.FSharp.Objects
+
+[<AbstractClass>]
+[<IterationCount(10)>]
+[<WarmupCount(3)>]
+[<Config(typeof<Configs.Matrix>)>]
+type Benchmarks(
+    buildFunToBenchmark,
+    converter: string -> float32,
+    binaryConverter,
+    buildMatrix)
+    =
+
+    let mutable funToBenchmark = None
+    let mutable matrix = Unchecked.defaultof<ClMatrix<float32>>
+    let mutable matrixPrepared = Unchecked.defaultof<PageRankMatrix<float32>>
+    let mutable matrixHost = Unchecked.defaultof<_>
+
+    let accuracy = 0.00000001f
```

Comment:
Still, why is it necessary separately? Perhaps we need to create a module with constants and actions on them in order to avoid mistakes in the future?

Abstract types could also be used here, so that they could be used only with the help of this most magical module.